Pre-processing/PlotAllChannels.py

```diff
@@ -1,21 +1,38 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from mne.io import concatenate_raws, read_raw_edf
+import os
 
 
 #Load Data
-load_path = 'C:/Users/User/Desktop/CSE 481C - Neruo Capstone/Project Repo/Data/TUH/00003010_s003_t000.edf'
+filename = '00003010_s003_t000'
+fileformat = '.edf'
+load_path = 'C:/Users/User/Desktop/CSE 481C - Neruo Capstone/Project Repo/Data/TUH/' + filename + fileformat
 print(load_path)
 raw = read_raw_edf(load_path, preload=True)
 
 #Inspect the .edf file
 print(raw.info)
 
+#Make a directory
+directory = filename + '_allChannels'
+parent_path = 'C:/Users/User/Desktop/CSE 481C - Neruo Capstone/Project Repo/Data/GeneratedPlots/'
+path = os.path.join(parent_path, directory)
+os.mkdir(path)
+
 #Plotting begins!
 #Raw data
-raw.plot(duration=100, block=True)
-plt.show()
+#todo: instead of scrolly image, get a static image
```

Comment:
If we get this running on the front end, we might not even need to save anything! Tricky problem though since browser only runs js/css/html